packages/form-layout/test/visual/lumo/form-layout-auto-responsive.test.js

```diff
@@ -152,4 +152,66 @@ describe('form-layout auto responsive', () => {
       await visualDiff(container, 'auto-rows-with-explicit-rows');
     });
   });
+
+  describe('colspan', () => {
+    beforeEach(async () => {
+      element = fixtureSync(
+        `
+          <vaadin-form-layout auto-responsive auto-rows max-columns="2">
+            <input placeholder="First name" />
+            <input placeholder="Last Name" />
+            <input placeholder="Email" />
+            <input placeholder="Phone" />
+            <input placeholder="Address" colspan="2" />
+          </vaadin-form-layout>
+
+          <style>
+            input {
+              justify-self: stretch;
+            }
+          </style>
+      `,
+        container,
+      );
+      await nextFrame();
+    });
+
+    it('default', async () => {
+      await visualDiff(container, 'colspan');
+    });
+  });
+
+  describe('colspan with explicit rows', () => {
+    beforeEach(async () => {
+      element = fixtureSync(
+        `
+          <vaadin-form-layout auto-responsive auto-rows max-columns="2">
+            <vaadin-form-row>
+              <input placeholder="First name" />
+              <input placeholder="Last Name" />
+            </vaadin-form-row>
+            <vaadin-form-row>
+              <input placeholder="Email" />
+              <input placeholder="Phone" />
+            </vaadin-form-row>
+            <vaadin-form-row>
+              <input placeholder="Address" colspan="2"  />
+            </vaadin-form-row>
+          </vaadin-form-layout>
+
+          <style>
+            input {
+              justify-self: stretch;
+            }
+          </style>
+      `,
+        container,
+      );
+      await nextFrame();
+    });
+
+    it('default', async () => {
```

Comment:
Let’s test this in both narrow and wide containers to verify that the colspan appears correctly adjusted in the screenshot.

Reply:
Tests added.